packages/console/compile.tsconfig.json

```diff
@@ -16,4 +17,4 @@
       "path": "../monaco/compile.tsconfig.json"
     }
   ]
-}
+}
```

Comment:
When working in Gitpod, Theia quite literally wouldn't let me restore this new line for some reason